### PR TITLE
Add test for users and posts controllers

### PR DESCRIPTION
**Pull request by Terbeche**
### I've made the following changes:
- Added RSpec as a gem to the project.
- Created a Request spec file for all controllers.
- Checked the following for each action:
     - If the response status was correct.
     - If a correct template was rendered.
     - If the response body includes correct placeholder text.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -240,6 +244,7 @@ DEPENDENCIES
   pg (~> 1.1)
   puma (~> 5.0)
   rails (~> 7.0.4)
+  rails-controller-testing
   rspec-rails
```
Comment (mateo951) [inline Gemfile.lock:248]: Has and uses RSpec gem for testing 😎

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
```
Comment (mateo951) [inline spec/requests/posts_spec.rb:2]: Has `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory 🔥

File: spec/requests/posts_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+RSpec.describe PostsController, type: :request do
```
Comment (mateo951) [inline spec/requests/posts_spec.rb:3]: Awesome! You used type: request as expected 👍

Comment (mateo951) [reply spec/requests/posts_spec.rb:3]: `index` and `show` actions for `User` and `Post` controllers are tested 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @Terbeche, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 3 above)
Comment (Terbeche) [reply spec/requests/posts_spec.rb:3]: Thank you so much for the positive feedback! 


than k_you

**Pull request merged**